ext/dapr-ext-fastapi/dapr/ext/fastapi/actor.py

```diff
@@ -48,23 +48,27 @@ def _wrap_response(
 
 
 class DaprActor(object):
-    def __init__(self, app: FastAPI):
-        self._dapr_serializer = DefaultJSONSerializer()
+    _DEFAULT_ACTOR_ROUTER_TAGS: List[str] = ['Actor']
+
+    def __init__(self, app: FastAPI,
+                 router_tags: Optional[List[str]] = None):
+        self._router_tags = router_tags if router_tags else self._DEFAULT_ACTOR_ROUTER_TAGS
```

Comment:
Almost.

If you don't want router tags you need to specify `[]`. `None` means you need to use the default so this will be:

```python
self._router_tags = self._DEFAULT_ACTOR_ROUTER_TAGS if router_tags is None else router_tags
```

Reply:
True 
Almost missed that ty

Reply:
I just changed this a little bit. You can provide either '[]' or 'None' to disable router tags going forward. Pushing that change now!